google/resource_google_project_test.go

```diff
@@ -417,6 +483,21 @@ resource "google_project" "acceptance" {
 }`, pid, pid, org)
 }
 
+func testAccProject_appEngineUpdate(pid, org string) string {
+	return fmt.Sprintf(`
+resource "google_project" "acceptance" {
+  project_id = "%s"
```

Comment:
nit: can you align the `=`s here? (as if you were to run terraform fmt)

Reply:
👍